NetKAN/KIS trad fr

```diff
@@ -0,0 +1,6 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "<KIS trad fr>",
```

Comment:
Identifier must start with a letter or number, and can have no spaces, eg `"KISTradFr"`